### PR TITLE
Views

### PR DESCRIPTION
**Pull request by BiancaMe**
-Create view for user index
-Create view for user shows
-Create view for posts index
-Create view for posts shows

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,17 @@
-<h1>List of Users </h1>
\ No newline at end of file
+<h1>List of Users </h1>
+
+<section id= "users">
+<% @users.each do |user| %>
+    <%= link_to user_path(user.id) do %>
+    <li class="user-item">
+        <div class = "user-img-container">
```
Comment (V-Blaze) [inline app/views/users/index.html.erb]: - You have done great so far :+1: :100: But there are a few requirements you need to meet before moving to the next milestone. :+1: :100: 

- - Screenshot reference:
Selection_053

- [x] According to the project requirement, you are meant to style your view files to look like the given wireframe. Currently, none of your view have been styled. I'll suggest you go to back to the project requirement, look at the wireframes, and style you pages to look like them :+1: :small_airplane: :wave: 

- - Wireframes:

Screenshot from 2023-12-02 13-00-20

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @BiancaMe 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Used created methods `recent_comments`✔
-  All view files were created👋 
-  Correct git flow✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -1 +1,18 @@
...
+            <div class="post-cont-container">
+                <span>Comments: <%= @post.comments_counter || 0 %> </span>
+                <span> Likes: <%= @post.likes_counter || 0 %></span>
+            </div>
+        </div>
+        <p><%= @post.text %></p>
+    </div>   
+    <div class="comments">
+        <p> <%= 'There is no comment yet' if @post.comments.length === 0 %></p>
+        <% @post.recents.each do |comment| %>
```
Comment (FrederickMih) [inline app/views/posts/show.html.erb:14]: - [Optional] @BiancaMe, I think you were supposed to display all comments on this template.

**Review by FrederickMih — APPROVED** [review]
## Hi :raising_hand: :raising_hand: @BiancaMe, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 

:tada: :champagne:  :tada: :champagne: 

## Highlights :green_circle: 

- PR is professional :+1: 
- Templates files have been styled as requested :+1: 

## Optional suggestions
**N/A**

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**